test/environmentSettings.ts

```diff
@@ -33,48 +35,48 @@ export class EnvironmentSettings {
   ];
   private _devHubAliasName = 'vscodeOrg';
   private _devHubUserName = 'svc_idee_bot@salesforce.com';
-  private _extensionPath = join(
-    __dirname,
-    '..',
-    '..',
-    'salesforcedx-vscode',
-    'packages'
-  );
-  private _startTime = new Date(Date.now()).toLocaleTimeString([],
-    { timeStyle: "short" }
-  );
+  private _extensionPath = join(__dirname, '..', '..', 'salesforcedx-vscode', 'packages');
+  private _startTime = new Date(Date.now()).toLocaleTimeString([], {
+    timeStyle: 'short'
+  });
   private _throttleFactor = 1;
+  private _devHubSfdxAuthUrl?: string = undefined;
 
-  private constructor() {}
+  private constructor() {
+    this._vscodeVersion =
+      process.env.VSCODE_VERSION || this._vscodeVersion;
+
+    if (process.env.SPEC_FILES?.length) {
+      this._specFiles = process.env.SPEC_FILES.split(',').map(
+        specFile => `./test/specs/**/${specFile.trim()}`
```

Comment:
@peternhale Just a heads up, with some of the work that @RitamAgrawal is doing, we might be changing the format of "SPEC_FILES"

As it is now, `process.env.SPEC_FILES` holds just the file name, assumes that the files are all in `./test/specs`, and then concatenates the strings.

Ritam is working on something where she might need a suite to be outside this directory location, so we might be changing the code from:
`'./test/specs/**/' + process.env.SPEC_FILES`
(and SPEC_FILES is foo.e2.ts)
...to:
`process.env.SPEC_FILES`
(and SPEC_FILES is whatever the env var is set to, like  ./test/specs/**/foo.e2.ts)